_employment/job-seekers/less-than-one-year.md

```diff
@@ -11,43 +11,81 @@ template: 1-topic-landing
 <div class="row" markdown="0">
 <div class="small-12 columns" markdown="1">
 
-Who should read this?
-•	Servicemembers, and their spouses, who are within one year of separating from the military
+### Who should read this?
 
-Why should you ready this? 
-•	You can take certain steps to improve your chances of moving smoothly and successfully into civilian life.
+Servicemembers, and their spouses, who are within one year of separating from the military
 
-What are the Steps?
+### Why should you ready this? 
 
-**1. 12 Months Before Leaving Active Duty** 
+You can take certain steps to improve your chances of moving smoothly and successfully into civilian life.
+
+### What are the Steps?
+
+</div>
+</div>
+
+
+<div class="small-12 columns" markdown="0">
+<ol class="process" markdown="0">
+<li class="step one wow fadeIn animated" markdown="0">
+
+<div markdown="1">
+
+
+#### 12 Months Before Leaving Active Duty
+
+</div>
+
+
+<div class="feature" markdown="1">
 
 - Fill in any gaps in your education or skill set by taking classes and obtaining certifications and licensures. The [GI Bill](/education/gi-bill/) may be used for some of this prior to exiting the service. 
 - Contact your local Transition Assistance Program (TAP) manager to schedule the mandatory Transition Goals, Plans, Success (GPS) curriculum. If you are unable to attend this training, it is available through [Joint Knowledge Online](https://jkodirect.jten.mil). You must complete the VA Benefits Briefings I and II, the Department of Labor Employment Workshop, and an Individual Transition Plan (ITP). ITPs are service specific, so check [documents](https://dodtap.mil/index.html) related to your branch. 
-- Review [military transcripts pivotal#110298578](/_dummy-placeholder.html) and service records for accuracy. 
-- Think about which industries and careers interest you, using [tools#110298582](/_dummy-placeholder.html) such as o*Net and CareerScope. 
-- Start developing a résumé by [translating military skills 110298580](/_dummy-placeholder.html) into equivalent civilian skills. Import these using the [Résumé Builder](/_dummy-placeholder.html). 
-- Clean up your personal social-media footprint and establish a presence on professional sites like LinkedIn. 
+- Review [military transcripts](employment/job-seekers/military-transcrips) and service records for accuracy. 
```

Comment:
@emilyville when I go back into github to fix this missing T, none of the changes I made are reflected. So should I wait a bit for it to reflect the changes I made?